experimental/LieAlgebras/src/PathModel.jl

```diff
@@ -0,0 +1,617 @@
+##### Kashiwara Crystal #####
+
+abstract type AbstractCrystal <: AbstractAlgebra.Set end
+
+abstract type AbstractCrystalElem <: AbstractAlgebra.SetElem end
+
+# AbstractCrystalElem required methods
+
+function Base.eps(b::AbstractCrystalElem)
+  error("not implemented")
+end
+
+function phi(b::AbstractCrystalElem)
+  error("not implemented")
+end
+
+@doc raw"""
+    Apply the root operator $\tilde e_i$ to `b` in place and return the result.
```

Comment:
```suggestion
    ealpha!(b::AbstractCrystalElem, i::Int)

Apply the root operator $\tilde e_i$ to `b` in place and return the result.
```
This and the next few docstrings are indented weirdly